docs/2_3_fmu_distribution.adoc

```diff
@@ -130,29 +130,26 @@ ABI version `<abi_ver>`::
 |Name
 |Description
 
-|160
-|Visual Studio 16.0 (2019)
-
-|150
-|Visual Studio 15.0 (2017)
+|80
+|Visual Studio 2005 (MSVC++ 8.0)
 
-|140
-|Visual Studio 14.0 (2015)
+|90
+|Visual Studio 2008 (MSVC++ 9.0)
 
-|120
-|Visual Studio 12.0 (2013)
+|100
+|Visual Studio 2010 (MSVC++ 10.0)
 
 |110
-|Visual Studio 11.0 (2012)
+|Visual Studio 2012 (MSVC++ 11.0)
 
-|100
-|Visual Studio 10.0 (2010)
+|120
+|Visual Studio 2013 (MSVC++ 12.0)
 
-|90
-|Visual Studio 9.0 (2008)
+|140
+|Visual Studio 2015 (MSVC++ 14.0)
 
-|80
-|Visual Studio 8.0 (2005)
+|150
```

Comment:
If this is the toolset version, it should be 141 for VS2017, or perhaps I am missing something?

Reply:
Fixed it. Thanks!